src/EditorFeatures/CSharpTest/Completion/CompletionProviders/CrefCompletionProviderTests.cs

```diff
@@ -551,6 +551,11 @@ public RefKind GetRefKindOfArgument(SyntaxNode node)
                 throw new NotImplementedException();
             }
 
+            public bool IsDeclaration(SyntaxNode node)
```

Comment:
not related to your change, but type name seems wrong. it is SyntaxFactsService not SemanticFactsService.